File: codeforces/791_A_BearAndBigBrother.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    int limak, bob, years = 0;
    cin >> limak >> bob;
    bool isHeavier = false;
    while (!isHeavier){
        limak = limak * 3;
        bob = bob * 2;
        if (limak > bob){
            isHeavier = true;
        }
        years = years + 1;
    }
    cout << years << "\n";
}

File: codeforces/791_A_BearAndBigBrother.go
package main

import "fmt"

func main() {
	var limak, bob int
	years := 0
	fmt.Scanf("%d %d", &limak, &bob)

	for {
		years = years + 1
		limak = limak * 3
		bob = bob * 2
		if limak > bob {
			break
		}
	}

	fmt.Println(years)
}